Exiled.API/Features/Player.cs

```diff
@@ -1864,6 +1864,11 @@ public void Broadcast(Broadcast broadcast, bool shouldClearPrevious = false)
             if (broadcast.Show)
                 Broadcast(broadcast.Duration, broadcast.Content, broadcast.Type, shouldClearPrevious);
         }
+        /// <summary>
+        /// Drops an item from the player's inventory.
+        /// </summary>
+        /// <param name="item">The item to be dropped.</param>
+        public void DropItem(Item item, bool isThrown = false) => Inventory.UserCode_CmdDropItem__UInt16__Boolean(item.Serial, isThrown);
```

Comment:
Will be ambiguous reference

Reply:
So what?

Reply:
Plugin will require recompile => breaking change. Default parameter's value should be removed